app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -166,7 +166,7 @@ class Scene extends React.Component {
     // directly on top of each other.
     World.add(
       this.engine.world,
-      Bodies.circle(this.state.width / 2 + (Math.random() - 0.5), -50, 8, {
+      Bodies.polygon(this.state.width * Math.random(), -50, 8, 8, {
```

Comment:
looks like more then just a shape change here. I imagine some parts of this belong in 387bd0ec52add4fdc36587810dca1446887bef7d